test cases/common/243 custom target feed/meson.build

```diff
@@ -0,0 +1,24 @@
+project('custom target feed', 'c')
+
+python3 = import('python3').find_python()
+
+# Note that this will not add a dependency to the compiler executable.
+# Code will not be rebuilt if it changes.
+comp = '@0@/@1@'.format(meson.current_source_dir(), 'my_compiler.py')
```

Comment:
Um... why are we doing this?

Reply:
This is copy-pasta from `109 custom target capture`.

Reply:
Yeah, that seems to be that way since it was added in 70d94a555004563178878b73274e345adff5b4e8.  I have no idea why :)